urban-os/values.yaml

```diff
@@ -79,3 +88,282 @@ kubernetes-data-platform:
       rootCertPath: /etc/ssl/certs/ca-certificates.crt
   minio:
     enable: false
+
+grafana:
+  adminUser: admin
+  # An adminPassword must be set
+  # adminPassword: {password}
+
+  datasources:
+    datasources.yaml:
+      apiVersion: 1
+      datasources:
+        - name: Prometheus
+          type: prometheus
+          url: http://urban-os-prometheus-server
```

Comment:
Will this always be prefaced with urban-os? Or is that the release name

Reply:
That is dependent on the release name

Reply:
@LtChae Is there a way to reference the release name dynamically?

Reply:
We've changed it so that it will always be prefaced instead of being dynamic, so I think this can be resolved 👍🏼

`fullnameOverride` in the prometheus server values let us call it `urban-os-prometheus-server`, no matter the release

Reply:
This was also accomplished for grafana